.github/workflows/ci.yaml

```diff
@@ -13,32 +13,10 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  Run-Unit-tests:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        python: [ '3.8', '3.9', '3.10', '3.11', '3.12' ]
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
-        with:
-          python-version: '${{ matrix.python }}'
-          architecture: 'x64'
-      - uses: actions/cache@v3
-        with:
-          path: |
-            ~/.cache/pip
-          key: ${{ runner.os }}-${{ hashFiles('setup.cfg') }}
-      - run: pip install -U pip wheel
-      - run: pip install -e '.[tests]'
-      - run: pytest --junit-xml=test-report/report.xml tests
 
   build-n-publish:
     if: github.event_name == 'release'
     name: Build and publish Python 🐍 distributions 📦 to PyPI
-    needs:
-      - Run-Unit-tests
```

Comment:
We would need to add this dependency back and all other tests so that we verify CI jobs pass before publishing a release.